src/libmongoc/tests/unified/entity-map.c

```diff
@@ -161,14 +161,12 @@ should_ignore_event (entity_t *client_entity, event_t *event)
       return true;
    }
 
-   if (!client_entity->ignore_command_monitoring_events) {
```

Comment:
Oh good catch! It looks like this returned early if a client entity was not specified with `ignoreCommandMonitoringEvents`